### PR TITLE
Hacktheburgh19 landing page changes

### PR DESCRIPTION
**Pull request by lukeocodes**

### CODE REVIEW

File: config/landing_pages/hacktheburgh19.yml [hunk 1]
```diff
@@ -52,8 +56,17 @@ page:
             name: "icon-laptop"
             color: blue
           text:
-          - content: Build an application that has a strong approach to accessibility, using the Nexmo Voice API. The Nexmo Voice API supports the ability to get a raw stream of audio from a phone call or conference. Use this and other features from the Voice API to make a feature more accessible in your hack. 
+          - content: Build an application that has a strong approach to accessibility, using the Nexmo Voice API. The Nexmo Voice API supports the ability to get a raw stream of audio from a phone call or conference.
```
Comment (leggetter) [inline config/landing_pages/hacktheburgh19.yml]: Over WebSocket?

Comment (leggetter) [reply config/landing_pages/hacktheburgh19.yml]: A bit more content here pointing to:

- https://developer.nexmo.com/extend/ibm-watson-speech-to-text
- https://developer.nexmo.com/extend/azure-speech-to-text
- https://developer.nexmo.com/extend/google-transcription
- https://developer.nexmo.com/extend/ibm-watson-sms-sentiment-analysis

The point here is that it provides relevant resources related to our challenge.

File: config/landing_pages/hacktheburgh19.yml [hunk 2]
```diff
@@ -52,8 +56,17 @@ page:
...
+
+    - column:
+      - type: structured_text
+        structured_text:
+          header: "Mini Challenge"
+          icon:
+            name: "icon-laptop"
+            color: blue
+          text:
           - content: A mini challenge with prizes is also running. Build an application to send us a text message using the Nexmo Messages API. Send NUMBER a text message and then come and see us, show us your code and then get a prize.
```
Comment (leggetter) [inline config/landing_pages/hacktheburgh19.yml]: Link to relevant resources? Messages API docs. Prizes are [BBC micro:bit](https://microbit.org/) and [Pixl.js](https://shop.espruino.com/pixljs). Including the links to what these are would be useful too.

**Comment by lukeocodes** [comment]
@leggetter more changes made! looks a bit better now

https://nexmo-developer-pr-1532.herokuapp.com/hacktheburgh2019

**Pull request merged**